### ISSUE TITLE
Ability to pass function to property defaults

### ISSUE THREAD

**Issue by vinerz** [brikteknologier/seraph-model#96]
Hello!

I'd wish to have a way to pass a function to default such as `Date.now`, so when the key is not set, Seraph fires it and set the key's value to its return value.

Thanks again,
Vinicius

**Comment by andyrichardson** [comment]
Yes, if you look [here](https://github.com/brikteknologier/seraph-model#default), you'll see that you can set a default value within your schema.
